cmd/rookflex/cmd/unmount.go

```diff
@@ -42,21 +42,44 @@ func handleUnmount(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("Rook: Error getting RPC client: %v", err)
 	}
+	mountDir := args[0]
+
+	log(client, fmt.Sprintf("unmounting mount dir: %s", mountDir), false)
+
+	mounter := getMounter()
+
+	// Check if it's a cephfs
+	err = mounter.Runner.Command("df", "--type", "ceph", mountDir).Run()
+	if err == nil {
+		// This is a cephfs. Unmount pod mount dir
```

Comment:
should this block be refactored to a unmountCephFS function like we did with mount?  may help with readability.

Reply:
I extracted the mount functions because it was getting big. Unmount was not as big but it doesn't hurt to do that.